app/transport/rest/contracts.go

```diff
@@ -4,6 +4,8 @@ import (
 	"github.com/go-devs-ua/octagon/app/entities"
 )
 
+//go:generate mockgen -source=./contracts.go -destination=./mock_usecase.go -package=rest
```

Comment:
There is an idea that it is better to rename the name "mock_usecase.go" to "mock_usecase_test.go". The word "test" will allow you not to add an unnecessary file when building

Reply:
done